lib/startup/loader_progress.js

```diff
@@ -0,0 +1,131 @@
+var PORT = 55000;
+var SPLASH = __dirname + '/index.htm';
+var __progressLog = [];
+var __messageLog = [];
+var __config = {};
+var __connections = {};
+var __listening = false;
+var http = require('http');
+
+function LoaderProgress(config){
+
+  if (!config)
+    config = {};
+
+  if (!config.port)
+    config.port = 55000;
```

Comment:
This initialisation sequence is a bit confusing and could be cleaner. It's not clear whether we are going to use PORT or config.port later on since both are being initialised:
```
PORT = 55000;
if (!config.port) config.port = 55000;
PORT = config.port;
```
If we're going to use PORT we should ignore config.port from this point on, so maybe it should rather read:
```
PORT = 55000;
if (config.port) PORT = config.port;
```